src/xpcc/container/deque_impl.hpp

```diff
@@ -36,8 +36,9 @@
 
 template<typename T, std::size_t N>
 xpcc::BoundedDeque<T, N>::BoundedDeque() : 
-	head(0), tail(1), size(0)
+	head(0), size(0)
```

Comment:
I'd like this still in the constructor list, that will likely result in one less constructor call for static instantiations.
```diff
-	head(0), tail(1), size(0)
+	head(0), tail(N == 1 ? 0 : 1), size(0)
```